astriaorg/dev-cluster
# chore: update versions + cleanup

- updates to new geth version + genesis version, migrating away from using script to start
- updates all astria component images
- adds ability to disable faucet and blockscout in rollup
- fixes issues with volumes for local storage
- some general cleanup of rollup chart

## Testing
- spin up sequencer cluster & execute faucet transaction against, validate result.
- spin up rollup, execute transaction, bring down and verify transaction syncs.
